logrotate-http.conf

```
@@ -0,0 +1,14 @@
+/var/log/httpd/*.log {
+    maxsize 10K
+    dateext
+    dateformat -%Y%m%d-%s
+    rotate 2
+    missingok
+    notifempty
+    compress
+    delaycompress
+    sharedscripts
+    postrotate
+        PID=$(pgrep -o httpd) && kill -USR1 $PID
+    endscript
```

```suggestion
    postrotate
        PID=$(pgrep -o httpd); if [ "$PID" ]; then kill -USR1 $PID; fi
    endscript
```
You should use `if` instead of `&&` here because the `&&` construction would return nonzero if the pgrep failed (i.e. no httpd running), which causes the logrotate script to fail.